interface/forms/newpatient/common.php

```diff
@@ -76,444 +76,501 @@ function sensitivity_compare($a, $b)
-     <td class='bold' nowrap><?php echo xlt('Onset/hosp. date:'); ?></td>
-     <td class='text' nowrap><!-- default is blank so that while generating claim the date is blank. -->
-      <input type='text' size='10' class='datepicker input-sm' name='form_onset_date' id='form_onset_date'
-       value='<?php echo $viewmode && $result['onset_date']!='0000-00-00 00:00:00' ? attr(oeFormatShortDate(substr($result['onset_date'], 0, 10))) : ''; ?>'
-       title='<?php echo xla('Date of onset or hospitalization'); ?>' />
-     </td>
-    </tr>
-    <tr>
-     <td class='text' colspan='2' style='padding-top:1em'>
-     </td>
-    </tr>
-   </table>
-
-  </td>
-
-
-  <td class='bold' width='33%' nowrap>
-
-<?php
-  // To see issues stuff user needs write access to all issue types.
-  $issuesauth = true;
-foreach ($ISSUE_TYPES as $type => $dummy) {
-    if (!acl_check_issue($type, '', 'write')) {
-        $issuesauth = false;
-        break;
-    }
+fieldset{
+    background-color: #F2F2F2;
+    margin-bottom:10px;
+    padding-bottom:15px;
 }
-
-if ($issuesauth) {
-?>
-  <div style='float:left'>
-<?php echo xlt('Issues (Injuries/Medical/Allergy)'); ?>
-  </div>
-  <div style='float:left;margin-left:8px;margin-top:-3px'>
-    <?php if (acl_check('patients', 'med', '', 'write')) { ?>
-       <a href="../../patient_file/summary/add_edit_issue.php" class="css_button_small link_submit enc_issue"
-        onclick="top.restoreSession()"><span><?php echo xlt('Add'); ?></span></a>
-        <?php } ?>
-  </div>
-<?php } ?>
-
-  </td>
- </tr>
-
- <tr>
-  <td class='text' valign='top'>
-   <textarea name='reason' cols='40' rows='12' wrap='virtual' style='width:96%'
-    ><?php echo $viewmode ? text($result['reason']) : text($GLOBALS['default_chief_complaint']); ?></textarea>
-  </td>
-  <td class='text' valign='top'>
-
-<?php if ($issuesauth) { ?>
-   <select class="form-control" multiple name='issues[]' size='8' style='width:100%'
-    title='<?php echo xla('Hold down [Ctrl] for multiple selections or to unselect'); ?>'>
+.btn-link:focus, .btn-link:hover {
+text-decoration: none ;
+}
+.btn-link{
+    border:1px solid #CCC;
+    border-radius: 3px !Important;
+}
+.btn-link.active,.btn-link.focus,.btn-link:active,.btn-link:focus,.btn-link:hover,.open>.dropdown-toggle.btn-link {
+    color:#333;
+    border-color:#ADADAD;
+    box-shadow:0px 0px 2px #ADADAD inset;
+    transition: border-color ease-in-out .3s,box-shadow ease-in-out .3s;
+}
+.btn-separate-left{
+    margin-left:20px !Important;
+}
+.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle) {
+    border-top-right-radius: 3px;
+    border-bottom-right-radius: 3px;;
+}
+.btn-group-pinch >.btn:nth-last-child(2):not(.dropdown-toggle) {
+    border-top-right-radius: 3px !Important;
+    border-bottom-right-radius: 3px !Important;
+}
+@media only screen and (max-width: 1024px) {
+    #visit-details [class*="col-"], #visit-issues [class*="col-"]{
+    width: 100%;
+    text-align:left!Important;
```

Comment:
Didn't we get rid of most of this embedded stuff already in prior stuff (like the forms)?

Reply:
@media rules are needed to keep page responsive, unfortunately due to the varying design of the pages the break point  differs for individual pages

Reply:
hi @zbig01 ,
I am referring to all the embedded styling above the @media section

Reply:
I can get rid of them if I can add a few oe- classes to style_light.css and style_manila.css. Would you like me to do so.

Reply:
hi @zbig01 , but none of these look like oe- classes and instead override the way buttons are viewed/separated (which we decided in past to leave up to the primary styles). Let me know if I am wrong on this assumption, though.

Reply:
for example, what happened to the use of oe-opt-btn-group-pinch and oe-opt-btn-separate-left classes?

Reply:
Sorry have been busy,will look into it and get back.